api/models.go

```diff
@@ -1,10 +1,24 @@
 package api
 
+import "github.com/mhghw/fara-message/db"
+
 type loginBody struct {
 	Username string `json:username`
 	Password string `json:password`
 }
 
-type HTTPError struct{
+type HTTPError struct {
 	Message string `json:message`
 }
+
+type NewGroupChat struct {
+	ChatName string    `json:"chatName"`
+	Users    []db.User `json:"users"`
+}
+type NewDirectChat struct {
+	Users []db.User `json:"users"`
+}
+
+type Chat struct {
+	ID int
+}
```

Comment:
Whats chat?
try adding more context to struct name
like GetChatRequest or GetChatBody